[2024-09-11T01:52:45.572+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-11T01:52:45.623+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v4.third_task scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2024-09-11T01:52:45.637+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v4.third_task scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2024-09-11T01:52:45.638+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-09-11T01:52:45.665+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): third_task> on 2022-08-29 00:00:00+00:00
[2024-09-11T01:52:45.678+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=5479) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T01:52:45.679+0000] {standard_task_runner.py:63} INFO - Started process 5489 to run task
[2024-09-11T01:52:45.677+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v4', 'third_task', 'scheduled__2022-08-29T00:00:00+00:00', '--job-id', '1264', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpoefgttjy']
[2024-09-11T01:52:45.680+0000] {standard_task_runner.py:91} INFO - Job 1264: Subtask third_task
[2024-09-11T01:52:45.772+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v4.third_task scheduled__2022-08-29T00:00:00+00:00 [running]> on host 0359316e7bd1
[2024-09-11T01:52:45.983+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='vohoang' AIRFLOW_CTX_DAG_ID='our_first_dag_v4' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2022-08-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-08-29T00:00:00+00:00'
[2024-09-11T01:52:45.994+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-11T01:52:46.022+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-11T01:52:46.024+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "hey, I am task3 and will be running after task 1"']
[2024-09-11T01:52:46.041+0000] {subprocess.py:86} INFO - Output:
[2024-09-11T01:52:46.045+0000] {subprocess.py:93} INFO - hey, I am task3 and will be running after task 1
[2024-09-11T01:52:46.046+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-11T01:52:46.047+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-11T01:52:46.093+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v4, task_id=third_task, run_id=scheduled__2022-08-29T00:00:00+00:00, execution_date=20220829T000000, start_date=20240911T015245, end_date=20240911T015246
[2024-09-11T01:52:46.145+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-11T01:52:46.171+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
